OpenSim/Examples/ControllerExample/ControllerExample.cpp

```diff
@@ -317,8 +318,11 @@ int main()
         manager.integrate( si );
 
         // Save the simulation results.
-        osimModel.printControlStorage( "tugOfWar_controls.sto" );
-        manager.getStateStorage().print( "tugOfWar_states.sto" );
+        auto controlsTable = osimModel.getControlsTable();
+        MOTFileAdapter::write(controlsTable, "tugOfWar_controls.mot");
```

Comment:
I would like us to avoid using `.mot` as our "output" format. We have total control over `.sto` but `.mot` was originally defined and used by SIMM (motion file) and if we want to maintain some compatibility with SIMM we shouldn't use it as our own format. Can you revert these to `.sto`?  We can make `.sto` stand  for standard table opensim/output or whatever.

I also don't mind if a `STOFileAdapter` is just an alias of an `MOTFileAdapter` for the time being. I don't think it will be long before these two become more distinct.